src/result-handler.ts

```diff
@@ -22,6 +22,8 @@ interface ResultHandlerParams<RES> {
   input: FlatObject | null;
   /** null in case of errors or failures */
   output: FlatObject | null;
+  /** can be empty: check presence of the required property using "in" operator */
+  options: FlatObject;
```

Comment:
Current consideration: should it be consistent with `input` and `output` that are `null` in case of error?

Reply:
or would it be excessive complication?

Reply:
decided to keep it as it is for now